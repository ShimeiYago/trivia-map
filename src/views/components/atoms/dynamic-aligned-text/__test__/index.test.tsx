import { shallow, ShallowWrapper } from 'enzyme';
import { DynamicAlignedText, Props } from '..';

let wrapper: ShallowWrapper<Props>;

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<DynamicAlignedText children="xxx" />);
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
