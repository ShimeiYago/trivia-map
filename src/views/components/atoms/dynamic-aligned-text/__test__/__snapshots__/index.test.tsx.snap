// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests basic 1`] = `
<ForwardRef(Grid)
  container={true}
  justifyContent="center"
>
  <ForwardRef(Grid)
    item={true}
  >
    <ForwardRef(Typography)>
      xxx
    </ForwardRef(Typography)>
  </ForwardRef(Grid)>
</ForwardRef(Grid)>
`;
